examples/pytorch_ignite_simple.py

```diff
@@ -37,7 +37,6 @@
 
 import optuna
 
-
```

Comment:
@keisuke-umezawa @hvy @HideakiImamura 
I think we had a coding style convention to put two blank lines after library imports, but I cannot find it in the [current guide](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions). Can we accept this change?

Reply:
Oh, I did not know that. We have existing code where there's a single blank line, e.g. similar to this case where variables are declared (e.g. logger objects) or conditions using `try_import`. I thought that the "two blank lines rule" stemmed from PEP-8 https://www.python.org/dev/peps/pep-0008/#blank-lines and that it's okay as long as we follow

> Surround top-level function and class definitions with two blank lines.

Reply:
@hvy Thank you for your comment.

>  I thought that the "two blank lines rule" stemmed from PEP-8 https://www.python.org/dev/peps/pep-0008/#blank-lines

I understand. I'd like to accept this change according to PEP-8.